topics/basics/testing_plugins/integration_tests/integration_tests_intro.md

```diff
@@ -0,0 +1,262 @@
+<!-- Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license. -->
+
+# Creating the first integration test
+
+## Why create integration tests?
+* Testing complex scenarios: Some scenarios, particularly UI interactions, cannot be effectively covered by unit tests alone.
+* Full product testing: Integration tests run against the complete product rather than isolated components. This helps identify issues that unit tests might miss, such as module interaction problems, classpath conflicts, and plugin declaration issues.
+* User story validation: Integration tests typically mirror real user scenarios, ensuring your plugin works reliably from start to finish.
+
+Despite common perceptions, integration tests aren't as difficult to create as you might think! Let's explore how to set them up.
+
+## Adding dependencies
+Integration testing framework consists of two main components:
+* **Starter**: Handles IDE configuration, test project setup, IDE startup, and output collection.
+* **Driver**: Provides additional functionality we'll explore in future blog posts.
+
+The Starter framework exclusively supports JUnit 5, as it leverages JUnit 5's extensions and specialized listeners that aren't available in JUnit 4.
+
+Add the following dependencies to your build.gradle.kts:
+```kotlin
+dependencies {
+    intellijPlatform {
+        //...
+        testFramework(TestFrameworkType.Starter)
```

Comment:
Please explain what dependencies it will add under the hood (in general, that it will add started, driver, anything else)?.